src/main/java/core/basesyntax/FruitTransaction.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class FruitTransaction {
+    private String operation;
```

Comment:
```suggestion
    private Operation operation;
```
давай створимо єнам для типу операції